README.md

```diff
@@ -1,15 +1,28 @@
-This project can be used to quickly setup Satellite 6.1 or 6.2 install with the backup Satellite data or change the hostname on an existing Satellite
+# Satellite-clone
+Satellite-clone contains multiple simple Ansible playbooks that can be used to do any of the following tasks:
+* Setup Satellite 6.1 or 6.2 install with the backup Satellite data.
+* Change the hostname on an existing Satellite install.
 
- * [Cloning a Satellite machine](#cloning-a-satellite-machine)
- * [Changing the hostname of a Satellite machine](#changing-the-hostname-of-a-satellite-machine)
+## Getting Started
+- Make sure the machine you are running satellite-clone from can connect to the machine that you are executing the playbooks on via paswordless ssh.
+- Install `ansible` package in your workstation. For RHEL boxes, [access to EPEL] (https://access.redhat.com/solutions/3358) is required.
 
-## Cloning a Satellite machine:
+    ```console
+     # yum install -y ansible
+    ```
+- git clone this project in your workstation.
 
-### Pre-requisites:
+    ```console
+     # git clone git@github.com:RedHatSatellite/satellite-clone.git
+    ```
+- Now click one one of the links below to proceed further:
```

Comment:
How about something like:

`Now you can proceed to any of the following tasks:`